README.md

```diff
@@ -2,7 +2,54 @@
 
 This repository contains formulae for multiple versions of dbt and supported adapters.
 
-### Development
+## Repo Structure
+
+---
+
+### Aliases
+
+    Contains a single file for every minor release starting with 1.0.
+
+These files define the most recent patch for a version when only given <major>.<minor>
+
+Example:  
+    Walking dbt-core 1.2 -> dbt-core 1.2.1
```

Comment:
I don't quite understand this example. What does "Walking" mean here? Also I would use an adapter as an example since dbt-core isn't an option on its own